### ISSUE TITLE
The app in the Vulkan mode flickered!

### ISSUE THREAD

**Issue by zwtderek1986** [hrydgard/ppsspp#19242]
### Game or games this happens in

None

### What area of the game / PPSSPP

Compile the iOS version of Master, switch the rendering engine after entering the APP into Vulkan and restart the app!

<img width="828" alt="image" src="https://github.com/hrydgard/ppsspp/assets/39580899/87317b0e-15e3-4a61-a174-d0015e637ed6">

<img width="777" alt="image" src="https://github.com/hrydgard/ppsspp/assets/39580899/9cc88984-a6b5-4d2a-ae4c-db3beaa691b7">

### What should happen

Compile the iOS version of Master, switch the rendering engine after entering the APP into Vulkan and restart the app!

### Platform

iOS

### Mobile device model or graphics card (GPU)

Apple A17 Pro GPU

### PPSSPP version affected

v1.17.1-681-g8600e81c62

### Last working version

v1.17.1-334-g1786a4ddb0

### Graphics backend (3D API)

Vulkan

### Checklist

- [X] Test in the [latest git build](https://www.ppsspp.org/devbuilds) in case it's already fixed.
- [X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [ ] Try resetting settings or older versions and include if the issue is related.
- [ ] Try without any cheats and without loading any save states.
- [ ] Include logs or screenshots of issue.

**Comment by hrydgard** [comment]
Yeah, the Vulkan loader is linked differently in the non-AppStore build and I haven't tested that path. needs work. I'll probably switch to static linking in this build too, should take care of it.

**Comment by zwtderek1986** [comment]
> Yeah, the Vulkan loader is linked differently in the non-AppStore build and I haven't tested that path. needs work. I'll probably switch to static linking in this build too, should take care of it.

Okay,  look forward to it!

**Comment by Halo-Michael** [comment]
@zwtderek1986 Main branch bundled an out of time dylib. Try this and check if it helps:
https://halo-michael.github.io/ipas/PPSSPP_0v1.17.1-714-g3444c32097.ipa

**Comment by zwtderek1986** [comment]
> @zwtderek1986主分支捆绑了一个过时的 dylib。尝试一下并检查它是否有帮助： https://halo-michael.github.io/ipas/PPSSPP_0v1.17.1-714-g3444c32097.ipa

Thanks!
